LawGuardPro.Application/Features/Settings/Profiles/ProfileEditCommand.cs

```diff
@@ -0,0 +1,39 @@
+﻿using LawGuardPro.Application.DTO;
+using MediatR;
+using System.Net;
+using System.Reflection;
+using AutoMapper;
+using LawGuardPro.Application.Common;
+using LawGuardPro.Application.Features.Identity.Interfaces;
+
+namespace LawGuardPro.Application.Features.Settings.Profiles;
```

Comment:
Always put a space between class declaration and namespace

Reply:
Done.